01-arch/07-acl.md

```diff
@@ -103,6 +103,34 @@ and can be presented in different intermediate formats, like JSON, for the users
 }
 ```
 
+Note that some filters with `$Object` prefix are not suitable for making denying rules on certain operations. There may be an undefined behaviour on some combinations of NeoFS verbs and object attributes when eACL is set.
+
+|$Object:       |   GET   |  HEAD   |   PUT   | DELETE  | SEARCH  |  RANGE  |RANGEHASH|
+|---------------|---------|---------|---------|---------|---------|---------|---------|
+|version        |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|objectID       |    v    |    v    |    v    |    v    |    -    |    v    |    v    |
+|containerID    |    v    |    v    |    v    |    v    |    v    |    v    |    v    |
+|ownerID        |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|creationEpoch  |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|payloadLength  |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|payloadHash    |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|objectType     |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|homomorphicHash|    v    |    v    |    v    |    -    |    -    |    -    |    -    |
+|// User headers|    v    |    v    |    v    |    -    |    -    |    -    |    -    |
```

Comment:
Why is '//' here?

Reply:
That's how it was written in [the issue](https://github.com/nspcc-dev/neofs-node/issues/949#issuecomment-956104749).
I think @alexvanin can respond to this comment.